### PR TITLE
Implement calendar

### PR DESCRIPTION
**Pull request by Alex4Max**
The calendar display can be viewed in two states: by month and week (the current month) with the render of the hard-code events.
* When 'add redux' issue will be merged, i'll change the structure of this PR

### CODE REVIEW

File: src/components/Event.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+class Event extends Component {
+  constructor (props) {
+    super();
+    this.state = {
+      className: props.className,
```
Comment (sleepwalky) [inline src/components/Event.js]: It's not necessary to assign props to state. Most likely this component can be converted to stateless component.

File: src/components/table/Button.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function Button (props) {
+  return (
```
Comment (sleepwalky) [inline src/components/table/Button.js]: It's a good pattern to destructure props:
`const { onClick, value } = props;`

File: src/components/Event.js (hunk 1 above)
Comment (Alex4Max) [reply src/components/Event.js]: yes, but i'll change this component in the next PR. This is just a preparation for the next step.

**Comment by sleepwalky** [comment]
Closes #5

**Pull request merged**